tests/unit/multi-currency/test-class-currency-switcher-widget.php

```diff
@@ -85,6 +85,16 @@ public function test_widget_renders_enabled_currencies_with_symbol_and_flag() {
 		$this->expectOutputRegex( '/<option value="EUR">🇪🇺 &euro; EUR<\/option>/' );
 	}
 
+	public function test_widget_renders_hidden_input() {
+		$_GET = [
+			'test_name'  => 'test_value',
+			'test_array' => [ 0 => [ 0 => 'test_array_value' ] ],
```

Comment:
Does `$_GET` support named array keys, do you know? Just wondering if we should add a test for cases like `[  'val1' => 123, 'val2' => 234 ]`.

Reply:
🤔 Not sure if understand you correctly here. Do you mean something like e91ab56? Is that the case. Yes, we can use named keys in `$_GET` also.

Reply:
Yup, that's exactly what I meant! Sorry if my wording wasn't clear. 🙂 

Thanks for adding a test for that! all LGTM! :shipit: